markhor_bringup/launch/camera_rearview.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<launch>
+    
+    <!-- MS Lifecam Cinema Camera -->
+    <node name="camera_rearview" pkg="usb_cam" type="usb_cam_node" output="screen">
+    <param name="video_device" value="/dev/video0" />
```

Comment:
It would be best to have an udev rule to specify which camera will be represented to which /dev/video0